strawberryfields/api/result.py

```diff
@@ -118,7 +118,15 @@ def state(self):
 
     def __repr__(self):
         """String representation."""
-        shots, modes = self.samples.shape
+        try:
+            shots, modes = self.samples.shape
+        except ValueError:
```

Comment:
Think this would work fine for now actually! :100: Would be great to add a test though.

Reply:
Thanks @antalszava. I've added a test now! You were a bit too quick (and I maybe tagged you for a review a bit too quickly). :laughing: